public/outreach.html

```diff
@@ -168,11 +171,11 @@ <h1 class="hyphenate border-bottom border-light" id="art">Art making</h1>
          </div>
          <div class="container mt-5 pt-5 justify-content-center">
          <div class="text-center lead">
-            <h1 id="follow">Follow our projects</h1>
+            <h2 class="h1" id="follow">Follow our projects</h2>
             <p>Follow us on our #scicomm platforms to get the latest updates, and to connect with the various communities we belong to: Alaskan Arctic communities, Alaska scientists, the LTER network.</p>
-            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank">Facebook</a></button>
-            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://twitter.com/ArcticLagoons" target="_blank">Twitter</a></button>
-            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank">The Lagoon Exchange newsletter</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://facebook.com/arcticlagoons" target="_blank" aria-label="open in new tab">Facebook</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://twitter.com/ArcticLagoons" target="_blank" aria-label="open in new tab">Twitter</a></button>
+            <button class="btn btn-link btn-outline-primary m-1"><a rel="external noopener" href="https://utexas.box.com/v/BLE-LTER-Kaktovik-newsletter" target="_blank" aria-label="open in new tab">The Lagoon Exchange newsletter</a></button>
```

Comment:
specify what you're opening in a new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5